paper.tex

```diff
@@ -245,8 +245,8 @@ \subsection{External expectations}
 \section{Vision}%
 \label{sec:vision}
 In the following, we describe our vision of central RSE units at research institutions in Germany.
-As these institutions include universities, other colleges, as well as large associations like Max-Planck, Helmholtz, Fraunhofer, or Leibniz,
-they show a wide variety in organisational structure as well as internal scientific diversity.
+Institutions is, for our purposes, a broad term including universities, other colleges, associations like Max-Planck, Helmholtz, Fraunhofer, or Leibniz, as well as other research performing organisations,
```

Comment:
Hm. Universities yes, but MPG, HGF, FhF are umbrella organisations. I think we should compare universities with research centres/institutions inside one of these umbrella organisations. So, to me an institutions would be University of Gotham as well as the Helmholtz Centre for Bat Control, but not the Helmholtz Association.

Reply:
Hmm, I wanted to find a formulation, that also could work for DLR and HIFIS within Helmholtz....